Doc/library/time.rst

```diff
@@ -708,6 +626,101 @@ The module defines the following functions and data items:
       ('EET', 'EEST')
 
 
+Clock ID Constants
```

Comment:
I would suggest adding a link of this section to appropriate members of the time module. For example, see https://docs.python.org/3/library/time.html#time.clock_gettime -- I would add something like

```rst
See :ref:`time-clock-id-constants` for a list of accepted values for *clk_id*.
```

to the time.clock_gettime() documentation.

My wording is probably terrible (it's 3 a.m. here and my brain doesn't work well now), but I think you get the idea :)

And don't forget to create a link for the section:

```rst
.. _time-clock-id-constants:

Clock ID Constants
------------------
```